PROPOSAL.md

```diff
@@ -0,0 +1,39 @@
+# DRAFT Proposal
+
+TODO: Remove draft label once everyone has confirmed this text.
+
+This proposal is for a new segregated multi-signature wallet, designed to
+transparently manage a sum of 5,000,000 DOGE. The DOGE held in this wallet
+will disburse rewards to developers of Dogecoin Core for work on all
+contributions, no matter how big or small.
+
+We propose to create a multi-signature wallet, operated by Dogecoin Core
+developers and members of the Dogecoin Foundation. The wallet shall be secured
+with a three out of five signature threshold, with the following custodians:
+
+* Michi Lumin (Dogecoin Core developer, Dogecoin Foundation Board)
+* Ross Nicoll (Former Dogecoin Core developer)
+* Patrick Lodder (Dogecoin Core developer)
+* Marshall Hayner (Dogecoin Foundation Board)
+* chromatic (Dogecoin Core developer)
+
+The wallet shall operate as follows:
+
+1. With every release of Dogecoin Core, 500,000 DOGE shall be distributed
```

Comment:
I agree in spirit with this, however I can imagine the possibility of a hotfix release containing only one or two commits (suppose that we discover a crash bug on startup on one of our platforms and can't find a workaround). I would not want to distribute 500k Doge to the maintainers who are probably the ones who had to create that commit because we're the ones who let that bug through.

I'm not sure what alternate wording might work better, though "Major or Minor release" might be fine; we could consider the scenario I proposed a "Patch release".

Reply:
I'm happy with/without threshold criteria, and 25 PR cut-off seems as good a line as any if we want a preset cut-off. Hoping to get comments from chromatic and Marshall on this before I update the text, though.

Reply:
Patrick's suggestion makes sense to me, and it catches the case where a patch release contains commits we want to credit to contributors for distribution purposes.

When we assign weights to commits, we can (and have in the past) exclude maintainer commits where we should have known better.

Reply:
Revised the text on that basis - I'll squash the commits down if there's general agreement, but wanted to keep the amendment in its own commit so it's clear what I've changed.

Reply:
Yes I think this is good as long as it remains criteria for payout roubd grouping and not determining what a release type is. Maybe some addendum that a patch release won't trigger a payout if say for some reason a 26th PR is raised at the last minute? Just some kind of edge case catcher; that 26th PR could be a whitespace revision for example...

Reply:
Not... sure how to usefully quantify that, I think generally there has to be a certain amount of applied sense. Sample text or even better a PR against this PR would be appreciated, though.

Reply:
I'm generally leaning toward "We cannot perfectly quantify this, and trying will make us crazy", but if there's a suggested number or revised text do let me know.